docs/src/main/asciidoc/security-authentication-mechanisms-concept.adoc

```diff
@@ -198,6 +198,7 @@ For more information about OIDC authentication and authorization methods you can
 |Bearer token authentication mechanism |xref:security-oidc-bearer-token-authentication-concept.adoc[OIDC Bearer authentication]
 |Authorization code flow authentication mechanism |xref:security-oidc-code-flow-authentication-concept.adoc[OpenID Connect (OIDC) authorization code flow mechanism]
 |Multiple tenants that can support bearer token or authorization code flow mechanisms |xref:security-openid-connect-multitenancy.adoc[Using OpenID Connect (OIDC) multi-tenancy]
+|Well-Known OpenID Connect Providers |xref:security-openid-connect-providers.adoc[Configuring Well-Known OpenID Connect Providers]
```

Comment:
Hi @sberyozkin,
I have just a couple of small comments here:
- For consistency, would it be possible to change the capitalization of the text and link text to sentence case rather than title case, that is, change to "Well-known OpenID Connect providers" and "Configuring well-known OpenID Connect providers"
- Also, just a question/thought about the use of the term "well-known". It is a somewhat informal term and perhaps subjective. Could we remove it or perhaps change to something like "Common"?